Admin_Registration/README.md

```diff
@@ -0,0 +1,64 @@
+# Admin Registration
+
+## Type
+
+Improper Input Validation
+
+## Description
+
+Register as a user with administrator privileges.
+
+## What do I need?
+
+- user modal
+- user registration

```

Comment:
this section should also list tools you may use, like burpsuite (maybe also with version hint if necessary)